t/tnt/init.lua

```diff
@@ -37,6 +45,114 @@ local function tnt_prepare(cfg_args)
     box.cfg(cfg_args)
 end
 
+local function tnt_cluster_prepare(cfg_args)
+    -- prepare master
+    cfg_args = cfg_args or {}
+    local files = fio.glob(fio.pathjoin(dir, '*'))
+    for _, file in pairs(files) do
+        if fio.basename(file) ~= 'tarantool.log' then
+            log.info("skip removing %s", file)
+            fio.unlink(file)
+        end
+    end
+
+    cfg_args['wal_dir']         = dir
+    cfg_args['read_only']       = false
+    cfg_args[snapdir_optname()] = dir
+    cfg_args[logger_optname()]  = fio.pathjoin(dir, 'tarantool.log')
+    cfg_args['listen']          = bind_master
+    cfg_args['replication']     = {'replicator:password@' .. bind_replica,
+                                   'replicator:password@' .. bind_master}
+    if vinyl_name() then
+        local vinyl_optname     = vinyl_name() .. '_dir'
+        cfg_args[vinyl_optname] = dir
+    end
+    cfg_args['replication_connect_quorum']  = 1
```

Comment:
Why do we need a quorum if we don't use `sync` spaces?

Reply:
I use `replication_connect_timeout` parameter in order to prevent master node stuck in waiting for replica.
This parameter requires that `replication_connect_quorum` is set. Otherwise I will get an error: `can't initialize storage: Incorrect value for option 'replication': failed to connect to one or more replicas`